### ISSUE TITLE
Russian language

### ISSUE THREAD

**Issue by Slawa79** [aj3423/SpamBlocker#51]
Please add the option to change the language of the application to the settings. Russian Russian interests me, of course, and I figured it out in English, but it would be more convenient in Russian.

**Comment by aj3423** [comment]
The app follows the system language, it's already using Russian on your phone, assuming your system language is Russian. But there is no Russian translation text yet, so it uses the fallback language which is English. I don't know Russian, I need people contribute to the translation.

**Comment by Slawa79** [comment]
Understood. Thank you for the answer.

**Comment by aj3423** [comment]
Now it supports choosing language, however the Russian text seems to be very wide, not sure how to adjust it yet, but I guess better than nothing.
![image](https://github.com/aj3423/SpamBlocker/assets/4710875/d92a179f-bac5-4d9b-ab9c-e412cfb28dc4)

**Comment by Slawa79** [comment]
Yes, I agree some words take up too much space. And I'm already used to English, I translated incomprehensible words in the translator "according to the image " and set up a spam blocker 😆

**Comment by Korb** [comment]
> Sława79 closed this as not planned on May 16, 2024.

Since the application is currently available in `Russian` as well, I believe the ticket status should be changed to `completed`.
